metricbeat/module/system/process_summary/process_summary.go

```diff
@@ -101,13 +103,13 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 		MetricSetFields: outMap,
 	})
 
-	return nil
+	return degradeErr
```

Comment:
Same here, non-fatal errors should probably populate the `Error` field when creating the `mb.Event{`.

Reply:
same as above.